File: Lab_10/BankApp/Models/Transaction.cs
namespace BankApp.Models;

public class Transaction
{
    public string Type { get; }
    public decimal Amount { get; }
    public DateTime Date { get; }

    public Transaction(string type, decimal amount)
    {
        Type = type;
        Amount = amount;
        Date = DateTime.Now;
    }
}

File: lab10/Bank Account Command Pattern/BankApp/src/Models/Transaction.java
package Models;

import java.util.Date;

public class Transaction {
    private String type;
    private double amount;
    private Date date;
    public Transaction(String type, double amount) {
        this.type = type;
        this.amount = amount;
        this.date =  new Date();
    }
    public String getType() {
        return type;
    }
    public double getAmount() {
        return amount;
    }
    public Date getDate() {
        return date;
    }
}
